dear-diary/src/App.css

```diff
@@ -28,11 +28,25 @@
   color: #61dafb;
 }
 
+.container {
```

Comment:
with MUI don't need to write CSS like this. use their sx prop or styled to create custom designs